src/types.ts

```diff
@@ -354,6 +354,36 @@ export interface CalculateMetaTransactionQuoteParams extends SwapQuoteParamsBase
 
 /** End /swap types */
 
+/** Start /rfqm types */
+export interface FetchIndicativeQuoteParams {
+    apiKey: string;
+    buyAmount?: BigNumber;
+    buyToken: string;
+    sellAmount?: BigNumber;
+    sellToken: string;
+    takerAddress?: string;
+}
+
+export interface FetchIndicativeQuoteResponse {
+    buyAmount: BigNumber;
+    buyTokenAddress: string;
+    price: BigNumber;
+    sellAmount: BigNumber;
+    sellTokenAddress: string;
+}
+
+export interface FetchFirmQuoteParams {
+    apiKey: string;
+    buyAmount?: BigNumber;
+    buyToken: string;
+    sellAmount?: BigNumber;
+    sellToken: string;
+    takerAddress?: string;
+}
+export interface FetchFirmQuoteResponse {}
+
```

Comment:
Empty interface? Does it extend the indicative response?

Reply:
Just a placeholder at the moment - but yes, a good chance it will ultimately extend the indicative quote response